project/.php_cs.dist

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * DO NOT EDIT THIS FILE!
+ *
+ * It's auto-generated by sonata-project/dev-kit package.
+ */
+
+$header = <<<'HEADER'
+This file is part of the Sonata Project package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+HEADER;
+
+$projectPath = (preg_match('/dev-kit\/project$/', __DIR__, $matches)) ? __DIR__.'/..' : __DIR__;
+$rules = [
+    '@Symfony' => true,
+    '@Symfony:risky' => true,
+    'braces' => [
+        'allow_single_line_closure' => true,
+    ],
+    'header_comment' => [
+        'header' => $header,
+        'location' => 'after_open',
+    ],
+    'no_extra_consecutive_blank_lines' => true,
+    'no_useless_return' => true,
+    'ordered_imports' => true,
+    'phpdoc_order' => true,
+    'ternary_to_null_coalescing' => true,
+];
+
+if (\PHP_VERSION_ID >= 50600 && \PHP_VERSION_ID < 70000) {
```

Comment:
AFAIK, we can use the migration below even with PHP7. Please remove `\PHP_VERSION_ID < 70000`